lib/CMakeLists.txt

```diff
@@ -31,6 +31,9 @@ set(CMAKE_INCLUDE_CURRENT_DIR ON)
 if(NOT DEFINED BOOST_ROOT)
     set(BOOST_ROOT ${PXR_USD_LOCATION})
 endif()
+if(WIN32 AND MAYAUSD_DEFINE_BOOST_DEBUG_PYTHON_FLAG)
+    set(Boost_USE_DEBUG_PYTHON ON)
+endif()
```

Comment:
Sorry, I'm not super-familiar with this, but do we explicitly *not* want to set Boost_USE_DEBUG_PYTHON on Linux or Mac? A similar snippet was added in plugin/al/CMakeLists.txt. Maybe this should be done in the top-level CMakeLists.txt? We don't/can't disable building Python bindings in maya-usd, so I think we know we're going to need to find Boost, right?

Reply:
Alternatively... if we DO only want it on windows, maybe we could have this in the msvcdefaults.cmake?  That way it would all be in one place.  Also - again, not really familiar with the details, but do we ever want to `_add_define(BOOST_DEBUG_PYTHON)` but not `set(Boost_USE_DEBUG_PYTHON ON)`?  If they should always be a pair, makes even more sense to have them grouped.

Reply:
Yeah I guess it shouldn't have the WIN32. The reason this is here is because it is actually a variable used in the FindBoost.cmake. So I wanted it to be set directly above where we are calling find_package(Boost).